build.sbt

```diff
@@ -6,8 +6,6 @@ name := "thrift-serializer"
 organization := "com.gu"
 scalaVersion := "2.13.12"
 
-ThisBuild / versionScheme := Some("semver-spec")
```

Comment:
We're removing this because `sbt-version-policy` sets this for us and we don't need to try to overrule it!

https://github.com/guardian/gha-scala-library-release-workflow/blob/add-configuration-guidence/docs/configuration.md#recommended-sbt-plugins